src/components/PromotionalCard.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import { useState, useEffect } from "react";
+
+export const CardType = {
+  Collaborate: "Collaborate",
+  Membership: "Membership",
+} as const;
+
+type CardValues = (typeof CardType)[keyof typeof CardType];
+
+interface Cardprops {
+  type: CardValues;
+}
+
+const getColor = (type: CardValues) => {
+  return type === CardType.Collaborate ? "#5069ED" : "#ED7950";
+};
+
+const imgSrc = (type: CardValues) => {
+  return type === CardType.Collaborate
+    ? "colaboratorcard.svg"
+    : "sóciocard.svg";
+};
+
+const info: Record<CardValues, { name: string; text: string[] }> = {
+  [CardType.Collaborate]: {
+    name: "colaborador",
+    text: [
+      "Podes juntar-te a estes departamentos e contribuir para fazer o teu núcleo funcionar.",
+      "Vem fazer parte desta equipa e ajuda a tornar o CeSIUM melhor todos os dias.",
+    ],
+  },
+  [CardType.Membership]: {
+    name: "sócio",
+    text: [
+      "Participa em todos os nossos eventos gratuitamente, e usufrui de muitos mais benefícios.",
+      "Podes juntar-te a estes departamentos e contribuir para fazer o teu núcleo funcionar.",
+    ],
+  },
+};
+
+const getRandomText = (type: CardValues): string | undefined => {
+  const texts =
+    type === CardType.Collaborate
+      ? info[CardType.Collaborate].text
+      : info[CardType.Membership].text;
+
+  return texts && texts.length > 0
+    ? texts[Math.floor(Math.random() * texts.length)]
+    : "";
+};
+
+const PromotionalCard = ({ type }: Cardprops) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const [text, setText] = useState("");
+  const color = getColor(type);
+
+  useEffect(() => {
+    setText(getRandomText(type) ?? "");
+  }, [type]);
+
+  const buttonClass = `material-symbols-outlined absolute right-0 top-0 flex h-10 w-10 items-center justify-center rounded-full bg-[#FFFFFF1A] text-lg text-white transition duration-300 hover:bg-white md:relative md:right-0 md:top-0 md:h-10 md:w-10 md:text-xl`;
```

Comment:
We can remove this line since we moved it to the actual `<button>` element in the other comment.

```suggestion
```